packages/launchpad/src/setup-actions.ts

```diff
@@ -123,6 +123,17 @@ export const setUpActionsForTasks = ( {
 					useCalypsoPath = false;
 					break;
 
+				case 'review_site':
+					action = () => {
+						// redirects to the site slug home page in a new tab
+						window.open( `https://${ siteSlug }`, '_blank' );
+						updateLaunchpadSettings( siteSlug, {
```

Comment:
It opened a new window for me with the site, but it didn't mark the task as complete.
Checking the request I received the following error:

```
checklist_statuses: "review_site is not a valid property of Object."
```

Maybe it's something I'm missing?

I'm sandboxed (wpcom `trunk` updated), with the jetpack in the correct branch, synced with my site, and in this branch for Calypso.

Reply:
Did you also apply the changes to your Atomic Site?

Reply:
As we discussed, the issue is that I needed to also sync the Jetpack changes with the wpcom sandbox. Now it works for me!
Thank you for your help to check it!